components/avatar/avatar.css

```diff
@@ -0,0 +1,29 @@
+@import "../../docSite/docs.css";
+
+.container__components {
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: var(--gap);
+}
+.avatar {
+  object-fit: cover;
+  border-radius: 50%;
+}
+
```

Comment:
can make the spacing between CSS classes consistent. Otherwise, good work! 👍

Reply:
Thanks for reviewing!